core/src/main/java/io/undertow/server/protocol/ajp/AjpServerResponseConduit.java

```diff
@@ -266,6 +268,40 @@ protected void queueCloseFrames() {
         queueFrame(null, buffer);
     }
 
+    private void queueRemainingBytes(final ByteBuffer src, final ByteBuffer[] buffers) {
+        List<PooledByteBuffer> pools = new ArrayList<>(4);
+
+        try {
+            PooledByteBuffer newPooledBuffer = pool.allocate();
```

Comment:
This buffer is leaking as can be seen here: https://github.com/undertow-io/undertow/runs/5754811783?check_suite_focus=true (windows failure)